README.md

```diff
@@ -112,24 +112,31 @@ source ./setup_shell.sh
 ### 3. Setting up vendors
 
 RAI is vendor-agnostic. Use the configuration in [config.toml](./config.toml) to set up your vendor of choice for RAI modules.
+You can skip the setup of optional modules for now. It is covered in the [Developer Guide](docs/developers_guide.md).
```

Comment:
What do you mean?

Reply:
Fixing the link. I think what is optional is well captured by the "Optional" in the module table below.